php/WP_Mock.php

```diff
@@ -546,4 +546,88 @@ public static function getDeprecatedMethodListener(): DeprecatedMethodListener
     {
         return static::$deprecatedMethodListener;
     }
+
+    /**
+     * Adds an expectation that an action should be removed.
+     *
+     * @param string $action the action hook name
+     * @param string|callable-string|callable|Type $callback the callable to be removed
+     * @param int|Type|null $priority the priority it should be registered at
+     *
+     * @return void
+     * @throws InvalidArgumentException
+     */
+    public static function expectActionRemoved(string $action, $callback, $priority = null) : void
```

Comment:
How about returning the expectation once set? incase it was needed to for later. As well as adding nullable-int data type for the `$priority` argument

```suggestion
     * @return Mockery\Expectation
     * @throws InvalidArgumentException
     */
    public static function expectActionRemoved(string $action, $callback, ?int $priority = null) : Mockery\Expectation
```